server/src/main/java/org/opensearch/search/internal/InternalSearchResponse.java

```diff
@@ -91,5 +109,16 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeOptionalBoolean(terminatedEarly);
         out.writeOptionalWriteable(profileResults);
         out.writeVInt(numReducePhases);
+        writeSearchExtBuildersOnOrAfter(out, searchExtBuilders);
+    }
+
+    private static List<SearchExtBuilder> readSearchExtBuildersOnOrAfter(StreamInput in) throws IOException {
```

Comment:
@austintlee could you please prepare the pull request to `main` with only these version changes, so we could merge backport and fix main BWC promptly, thank you.